content/applications/websites/livechat/ratings.rst

```diff
@@ -2,44 +2,138 @@
+The page will list out the names of the :guilabel:`Live Chat Channels` whose ratings pages have been
+published. Next to the channel name is a speech bubble icon, which users can click on to be taken to
+the ratings' page for the respective channel.
+
+.. image:: ratings/live-chat-published-icon.png
+   :align: center
+   :alt: View of the webpage for Live Chat ratings emphasizing the channel icon.
+
+Make any desired changes or additions to this page, then click save on the :guilabel:`Webpage
+Editor`.
+
+To publish the `livechat` webpage, return to the list of webpages by navigating to
+:menuselection:`Site --> Pages`. Click the checkbox to the left of `livechat` in the list of pages
+to select the page. Then, click the checkbox under the column labeled :guilabel:`Is Published`. Then
+click :guilabel:`Save`.
+
+.. image:: ratings/live-chat-is-published.png
    :align: center
-   :alt: View of a chat window from an operator’s side highlighting a rating for Odoo Live Chat
+   :alt: View of the list of pages for a website with the 'is published' box emphasized.
+
+Once the page has been added to the site, ratings are set to be published by default. However,
+individual ratings can be manually selected to be hidden from the public. The rating will still be
+included in internal reports, and can still be viewed by internal teams. However, public website
+visitors and portal users will not have access.
+
+See :ref:`Hide individual ratings <livechat/overview/hide-ratings>` for more information.
+
+Customer ratings report
+=======================
 
-And under :menuselection:`Report --> Customer Ratings`.
+The :guilabel:`Customer Ratings` report (:menuselection:`Live Chat --> Report --> Customer Ratings`)
+displays an overview of the ratings received on individual support tickets, as well as any
+additional comments submitted with the rating.
 
-.. image:: ratings/customer_ratings.png
+.. image:: ratings/live-chat-ratings-report.png
    :align: center
-   :alt: View of the customer ratings page in Odoo Live Chat
+   :alt: View of the customer ratings report in Odoo Live Chat.
+
+Click on an individual rating to see additional details about the conversation, and the rating.
+
+.. _livechat/overview/hide-ratings:
+
+Hide individual ratings
+-----------------------
 
-Make the rating public
-======================
+Ratings are set to be published by default. However, individual ratings can be manually selected to
+be hidden from the public. The rating will still be included in internal reports, and can still be
+viewed by internal teams. However, public website visitors and portal users will not have access.
 
-Go to :menuselection:`Website --> Configuration --> Settings`, access your channel form, click on
-*Go to Website* and on *Unpublished*, to publish the rating of that channel on your website.
+To hide a rating, go to :menuselection:`Live Chat --> Reports --> Customer Ratings`. Click on the
```

Comment:
```suggestion
To hide a rating, go to :menuselection:`Live Chat app --> Reports --> Customer Ratings`. Click on the
```

This was another note someone gave me that I started to use, which was specify when the item is an app or not, so most things would start with the X app, then the rest. Just a thought- not necessary but it does add clarification (especially since we jump to the website app in this doc, too).